sys/linux/socket_netlink_route.txt

```diff
@@ -906,6 +907,8 @@ lwtunnel_policy [
 	LWTUNNEL_IP6_FLAGS	nlattr[LWTUNNEL_IP6_FLAGS, int16]
 	LWTUNNEL_IP6_OPTS	nlattr[LWTUNNEL_IP6_OPTS, ip_opts_policy]
 
+#	SEG6_IPTUNNEL_SRH	nlattr[SEG6_IPTUNNEL_SRH, ipv6_sr_hdr]
```

Comment:
I would suggest adding the TODO inline here. That way we can proceed to commit this to syzkaller without having to wait for the kernel (which can take months).

Reply:
Fixed.